style.css

```diff
@@ -0,0 +1,95 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Montserrat&family=Open+Sans:wght@400;500;600;700&family=Oswald&family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  background-color: #e4e5e7;
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  z-index: 9999;
+}
+
+.homepage {
+  background-image: url(Assets/files/header-background-mobile.jpg);
+  min-width: 375px;
+  border-bottom-left-radius: 10%;
+  margin-top: -1rem;
+  margin-left: -0.5rem;
+}
+
+#navbar {
+  z-index: 100;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 2rem;
+}
```

Comment:
- [ ] Kindly remember to add `display: 'flex'` to your header property as without it, your alignment and usage of flex won't work.

you can also add `align-items: 'center'` to center the alignment of the logo and hamburger 👍🏽